src/Umbraco.Infrastructure/Logging/Serilog/LoggerConfigExtensions.cs

```diff
@@ -17,69 +17,6 @@ namespace Umbraco.Extensions
 {
     public static class LoggerConfigExtensions
     {
-        /// <summary>
-        /// This configures Serilog with some defaults
-        /// Such as adding ProcessID, Thread, AppDomain etc
-        /// It is highly recommended that you keep/use this default in your own logging config customizations
-        /// </summary>
-        [Obsolete("Please use an alternative method. This will be removed in Umbraco 13.")]
```

Comment:
I don't think you can remove these as they are public classes.  They should have gone for an earlier major but now we'll need to keep for removal in 17.

You could remove them if you add suppressions, but we should only do that if we could be sure they aren't being used.